dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -86,7 +86,7 @@ ANNO_CLASSSOURCE_RESOURCE_NOTFOUND.useraction=Verify that the resource path for
 # # {4} the name of the class
 ANNO_CLASSSOURCE_CLOSE3_EXCEPTION=CWWKC0007W: [ {0} ]: The close of [{1}] as [{2}] under root [{3}] for class [{4}] failed.
 ANNO_CLASSSOURCE_CLOSE3_EXCEPTION.explanation=An attempt to close the InputStream for the class caused an exception.
-ANNO_CLASSSOURCE_CLOSE3_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CLOSE3_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.